CONTRIBUTING.md

```diff
@@ -163,123 +140,62 @@ description
-These are integration tests, making sure public API methods and events work as expected.
+- Tests are currently written with jest
 
 - To run all tests:
 
 ```bash
-npm run test
+npm run test:unit
 ```
 
-- To run all tests in Chromium
-```bash
-npm run ctest # also `ftest` for firefox and `wtest` for WebKit
-```
-
-- To run a specific test, substitute `it` with `it.only`, or use the `--grep 'My test'` CLI parameter:
-
-```js
-...
-// Using "it.only" to run a specific test
-it.only('should work', async ({server, page}) => {
-  const response = await page.goto(server.EMPTY_PAGE);
-  expect(response.ok).toBe(true);
-});
-// or
-playwright test --config=xxx --grep 'should work'
-```
-
-- To disable a specific test, substitute `it` with `it.skip`:
-
-```js
-...
-// Using "it.skip" to skip a specific test
-it.skip('should work', async ({server, page}) => {
-  const response = await page.goto(server.EMPTY_PAGE);
-  expect(response.ok).toBe(true);
-});
-```
-
-- To run tests in non-headless (headed) mode:
-
-```bash
-npm run ctest -- --headed
-```
-
-- To run tests with custom browser executable, specify `CRPATH`, `WKPATH` or `FFPATH` env variable that points to browser executable:
-
-```bash
-CRPATH=<path-to-executable> npm run ctest
-```
-
-- To run tests in slow-mode:
+or
 
 ```bash
-SLOW_MO=500 npm run wtest -- --headed
+yarn test:unit
 ```
 
-- When should a test be marked with `skip` or `fail`?
-
-  - **`skip(condition)`**: This test *should ***never*** work* for `condition`
-    where `condition` is usually a certain browser like `FFOX` (for Firefox),
-    `WEBKIT` (for WebKit), and `CHROMIUM` (for Chromium).
-
-    For example, the [alt-click downloads test](https://github.com/microsoft/playwright/blob/471ccc72d3f0847caa36f629b394a028c7750d93/test/download.spec.js#L86) is marked
-    with `skip(FFOX)` since an alt-click in Firefox will not produce a download
-    even if a person was driving the browser.
-
-
-  - **`fail(condition)`**: This test *should ***eventually*** work* for `condition`
-    where `condition` is usually a certain browser like `FFOX` (for Firefox),
-    `WEBKIT` (for WebKit), and `CHROMIUM` (for Chromium).
-
-    For example, the [alt-click downloads test](https://github.com/microsoft/playwright/blob/471ccc72d3f0847caa36f629b394a028c7750d93/test/download.spec.js#L86) is marked
-    with `fail(CHROMIUM || WEBKIT)` since Playwright performing these actions
-    currently diverges from what a user would experience driving a Chromium or
-    WebKit.
-
 ## Contributor License Agreement
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+This project welcomes contributions and suggestions. Most contributions require you to agree to a
 Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
 the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
```

Comment:
@0xLienid good catch. Yes, let's link to here https://github.com/GuardianUI/GuardianTest/blob/main/CLA.md